Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -200,6 +200,80 @@ public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
     }
 }
 
+/// An GRU cell.
+public struct GRUCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
+    public var updateWeight, updateWeight2: Tensor<Scalar>
```

Comment:
One last thing: I think it's better to rename variables that have a `2` variant to have a `1` suffix. So `updateWeight1`, `resetWeight1`, and `outputWeight1`. What do you think?

Reply:
I don't feel strongly at all, so I made the change.

Reply:
Sorry, I meant updating `updateWeight` to be `updateWeight1` so that you'll have `updateWeight1` and `updateWeight2`.

Reply:
Haha that makes way more sense. I agree this is an improvement. Made the changes accordingly

Reply:
Thanks!